src/d3d12/visualizer.rs

```diff
@@ -77,9 +75,18 @@ impl AllocatorVisualizer {
         self.color_scheme = color_scheme;
     }
 
-    fn render_main_window(&mut self, ui: &imgui::Ui, alloc: &Allocator) {
-        imgui::Window::new("Allocator visualization")
-            .collapsed(true, Condition::FirstUseEver)
+    fn render_main_window(&mut self, ui: &imgui::Ui, opened: Option<&mut bool>, alloc: &Allocator) {
+        let mut window = imgui::Window::new("Allocator visualization");
+
+        if let Some(opened) = opened {
+            if *opened {
+                window = window.opened(opened);
+            } else {
+                return;
+            }
+        }
```

Comment:
For general use, I think the precedent should be that you don't call this function if you don't want the UI to be rendered.

```suggestion
        if let Some(opened) = opened {
            window = window.opened(opened);
        }
```